### DISCUSSION TITLE
How can I make all profiles private automatically on startup?

### DISCUSSION THREAD

**Discussion by SosseHD** [netblue30/firejail#5798] [Q&A]
I tried to put "private" into the discord config. in the private section according to the profile template, but discord would not start. If i put the private flag when manually firejailing in the terminal, it works. how can I make this automatically?
I plan on enabling firecfg, and id like to have everything on private e.g. 
Can i add this to firejail.config?

which brings me to the next question, things in firejail.config, are they going to be prioritized over profiles?
thx

**Comment by rusty-snake** [comment]
https://github.com/netblue30/firejail/wiki/Creating-overrides

**Reply by SosseHD** [reply]
but why isnt it working when i type "private" into my discord profile?

**Reply by rusty-snake** [reply]
IDK. Providing basic debugging information could be helpful. Starting with an possible error. Sometimes program have bugs and crash if directories like `~/.config` do not exist.

**Comment by rusty-snake** [comment]
> Can i add this to firejail.config?

This is a configuration file defining how firejail behaves (some defaults and feature selection). **It is not a profile.** All options are documented in the file itself.

> things in firejail.config, are they going to be prioritized over profiles?

Yes. Note that default defined in it can of course be overridden.
